com.unity.netcode.gameobjects/Tests/Editor/NetworkManagerMessageHandlerTests.cs

```diff
@@ -207,7 +209,9 @@ public void MessageHandlerReceivedMessageServerClient()
     }
 
     // Should probably have one of these for more files? In the future we could use the SIPTransport?
+#if UNITY_EDITOR
```

Comment:
This one still feels weird/unrelated to the rest of these changes. Is it really necessary?